src/utils/constants.js

```diff
@@ -25,6 +25,7 @@ const EXECUTION_DOES_NOT_EXIST = 'ExecutionDoesNotExist';
 const ACCESS_DENIED = 'AccessDeniedException';
 
 const PUBLIC_ACCESS_ID = '00000000-0000-0000-0000-000000000000';
+const BIOMAGE_DOMAIN_NAMES = ['scp.biomage.net', 'scp-staging.biomage.net'];
```

Comment:
these are currently only used for checking if Biomage's privacy policy has been accepted. I would not really consider these as general purpose constants and should probably be in a domain specific file